test/selenium/OneOffContributionsSpec.scala

```diff
@@ -20,6 +20,41 @@ class OneOffContributionsSpec extends FeatureSpec with GivenWhenThen with Before
 
   feature("Sign up for a one-off contribution") {
 
+    scenario("One-off contribution sign-up with Stripe - GBP") {
+
+      val expectedPayment = 50.00
+      val expectedPaymentAsString = "50.00"
+      val currency = "AUD"
+      val testUser = new TestUser(driverConfig)
+      val landingPage = ContributionsLanding("au")
+      val oneOffContributionForm = OneOffContributionForm(testUser, expectedPayment.toInt, currency)
+      val oneOffContributionThankYou = new OneOffContributionThankYou
+
+      Given("that a test user goes to the contributions landing page")
+      goTo(landingPage)
+      assert(landingPage.pageHasLoaded)
+
+      When("he/she selects to make a one-time contribution")
```

Comment:
As we're adding a new test here, what about interacting with the amount picker (or entering a custom amount) so that we gain some coverage?

Reply:
Feel free to address that in a separate PR if you decide it's worth doing...

Reply:
Great idea! I've added a check for the enter-email page that it is displaying the correct amount. This is the only place the amount is visible.